panel/widgets/tables.py

```diff
@@ -1278,7 +1281,7 @@ def _stream(self, stream, rollover=None, follow=True):
         if self.pagination == 'remote':
             length = self._length
             nrows = self.page_size
-            max_page = length//nrows + bool(length%nrows)
+            max_page = length//nrows + bool(length%nrows) or 1
```

Comment:
Sorry for coming back to this so late? Is the logical grouping here correct? Doesn't quite look like it, because `True or 1` and `False or 1` both evaluate to 1. Should this be:

```suggestion
            max_page = (length//nrows + bool(length%nrows)) or 1
```

Reply:
I think this is a case of being too fancy. I will replace this with  `max(length//nrows + bool(length%nrows), 1)`, which makes it easier to read and understand. 

For the current implementation, I don't think parenthesis is necessary to get the desired effect, e.g., `1 + 1 or 1` equals 2, which is what we want.

Reply:
Hmm, not sure I follow, `True/False or 1` will always be equal to 1 so what is the point of even including the `bool(length%nrows)` condition.

Reply:
Bad example...

The `+` operator has higher precedence than `or` [ref](https://www.programiz.com/python-programming/precedence-associativity).

![image](https://user-images.githubusercontent.com/19758978/183613807-ee8ac79c-936a-4334-a37c-a0eedcd691fb.png)

Reply:
Thanks for your patience 🤦